### ISSUE TITLE
Documentation detection with @objc

### ISSUE THREAD

**Issue by jeeftor** [realm/SwiftLint#631]
<img width="463" alt="screen shot 2016-04-19 at 4 56 07 pm" src="https://cloud.githubusercontent.com/assets/6491743/14655300/a615eb52-064f-11e6-9f0d-faa7138ef7d4.png">

This detected correctly

```
@objc(DMLocationMessage) final public class LocationMessage: ParsedMessage {
```

This

```
@objc(DMLocationMessage)
final public class LocationMessage: ParsedMessage {
```

does not

**Comment by jpsim** [comment]
Could you please try again using #633? I just made some related fixes in SourceKitten and hope they address your issue.

**Comment by marcelofabri** [comment]
Closing this since we removed this rule.
